src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
```

Comment:
Also you can use this selector .stars__star:not(:first-child) { margin-left: 4px; } to shorten your code a bit